packages/neo-one-node-vm/src/stackItem/ArrayLikeStackItem.js

```diff
@@ -55,10 +59,13 @@ export default class ArrayLikeStackItem extends StackItemBase {
     throw new InvalidValueBufferError();
   }
 
-  toContractParameter(): ContractParameter {
-    return new ArrayContractParameter(
-      this.value.map((val) => val.toContractParameter()),
-    );
+  toContractParameter(seen: Set<StackItemBase> = new Set()): ContractParameter {
```

Comment:
the interface in the base class needs to be updated to include this extra argument.